src/main/resources/com/google/api/codegen/py/standalone_sample.snip

```diff
@@ -11,6 +12,10 @@
       @# To install the latest published package dependency, execute the following:
       @#   pip install {@sampleFile.gapicPackageName}
 
+      @# sample-metadata
+      @#   title: {@sample.title}
+      @#   description: {@sample.description}
+      @#   usage: python3 {@sampleFile.outputPath} {@commandlineArgmentList(sample.sampleInitCode.argDefaultParams)}
```

Comment:
Is it necessary to explicitly call out `python3` here?

Reply:
Our code assumes python3, right? So if we don't have a `#!/usr/bin/env python3` at the top, we should. If we do have that hash-bang, I don't have a strong opinion (slightly leaning towards leaving it in for explicitness)